CribblyBackend/Migrations/20210102_AddGameTable.cs

```diff
@@ -0,0 +1,29 @@
+using FluentMigrator;
+
+namespace CribblyBackend.Migrations
+{
+    [Migration(20210102072700)]
+    public class AddGameTable : Migration
+    {
+        public override void Down()
+        {
+            Delete.Table("Assignments");
+            Delete.Table("Games");
+        }
+
+        public override void Up()
+        {
+            Create.Table("Games")
+                .WithColumn("Id").AsInt32().PrimaryKey().Identity()
+                .WithColumn("GameRound").AsInt32().NotNullable()
+                .WithColumn("Type").AsString(255).NotNullable()
```

Comment:
It `Type` differentiating between prelims and torunament games? If so, we probably don't need to store it because `GameRound` tells us the same thing

Reply:
Taken out!

Reply:
@adamszcz I think you'll want to revert this migration and create a new one deleting the column; changes to this migration won't cause it to rerun.